framework/wcf/Extensibility/Transport/AzureQueueStorageClient/AzureQueueStorageClient/SampleProfileAzureQueueStorageBinding.cs

```diff
@@ -0,0 +1,434 @@
+﻿//----------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.  All rights reserved.
+//----------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Configuration;
+using System.ServiceModel;
+using System.ServiceModel.Channels;
+
+namespace Microsoft.Samples.AzureQueueStorage
+{
+
+    /// <summary>
+    /// Binding for Udp. This is our "sample profile" for Udp, which uses Text+Soap 1.2 
+    /// and allows for variation in Reliability capabilities. If ReliableSessionEnabled is set
+    /// then we will layer RM+CompositeDuplex on top of Udp. Otherwise we will just
+    /// have Udp on our stack.  
+    /// </summary>
+    public class SampleProfileAzureQueueStorageBinding : Binding
+    {
+        bool reliableSessionEnabled;
+
+        // private BindingElements
+        //CompositeDuplexBindingElement compositeDuplex;
+        ReliableSessionBindingElement session;
+        AzureQueueStorageTransportBindingElement transport;
+        MessageEncodingBindingElement encoding;
+
+        public SampleProfileAzureQueueStorageBinding()
+        {
+            Initialize();
+        }
+
+        public SampleProfileAzureQueueStorageBinding(bool reliableSessionEnabled) : this()
+        {
+            this.ReliableSessionEnabled = reliableSessionEnabled;
+        }
+
+        /*public SampleProfileUdpBinding(string configurationName) : this()
+        {
```

Comment:
Remove commented out constructor as only needed if we had config support.

Reply:
got it